### PR TITLE
contrib/razor: new package (2.86)

### PR DESCRIPTION
**Pull request by a16bitsysop**

### CODE REVIEW

**Comment by a16bitsysop** [comment]
This builds locally, I get the same error with 2.85 locally but not 2.86

**Comment by a16bitsysop** [comment]
fixed after removing perl-getopt-long which is now bundled with perl

File: contrib/razor/template.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "gmake",
+    "perl",
+]
+makedepends = ["perl"]
+depends = ["perl", "perl-uri"]
+pkgdesc = "Distributed, collaborative spam detection and filtering network"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "Artistic-2.0"
+url = "https://razor.sourceforge.net/index.html"
+source = f"https://cpan.metacpan.org/authors/id/T/TO/TODDR/Razor2-Client-Agent-{pkgver}.tar.gz"
```
Comment (q66) [inline contrib/razor/template.py:15]: this should use CPAN_SITE like other perl modules

File: contrib/razor/template.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+pkgname = "razor"
```
Comment (q66) [inline contrib/razor/template.py:1]: should likely be called perl-razor2-client-agent

Comment (q66) [reply contrib/razor/template.py:1]: that is, unless the perl module is secondary and the primary thing is the frontend tools

File: contrib/razor/template.py (hunk 1 above)
Comment (a16bitsysop) [reply contrib/razor/template.py]: okay

File: contrib/razor/template.py (hunk 2 above)
Comment (a16bitsysop) [reply contrib/razor/template.py]: some distros call it razor and some call it that:
https://repology.org/project/perl:razor2-client-agent/versions

Comment (a16bitsysop) [reply contrib/razor/template.py]: changing the name brings back the previous build failure?

Comment (a16bitsysop) [reply contrib/razor/template.py]: fixed it with another upstream patch, was parallel make

Comment (q66) [reply contrib/razor/template.py]: i still don't really understand how this is supposed to be used

so it is a perl module, is it a program, or what

File: contrib/perl-razor2-client-agent/template.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+hostmakedepends = [
+    "gmake",
+    "perl",
+]
+makedepends = ["perl"]
+depends = ["perl", "perl-uri"]
+pkgdesc = "Distributed, collaborative spam detection and filtering network"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "Artistic-2.0"
+url = "https://razor.sourceforge.net/index.html"
```
Comment (q66) [inline contrib/perl-razor2-client-agent/template.py:14]: website is dead?

Comment (nekopsykose) [reply contrib/perl-razor2-client-agent/template.py:14]: works for me, redirects to https://sourceforge.net/projects/razor/

Comment (q66) [reply contrib/perl-razor2-client-agent/template.py:14]: gets me a white page

File: contrib/razor/template.py (hunk 2 above)
Comment (a16bitsysop) [reply contrib/razor/template.py]: it is a program written in perl, it installs in user bin:
razor-admin   razor-check   razor-client  razor-report  razor-revoke

but also installs perl modules in /usr/lib/perl5 that these use

File: contrib/perl-razor2-client-agent/template.py (hunk 3 above)
Comment (a16bitsysop) [reply contrib/perl-razor2-client-agent/template.py]: I changed to the redirect address https://sourceforge.net/projects/razor

File: contrib/razor/template.py (hunk 2 above)
Comment (q66) [reply contrib/razor/template.py]: if it's primarily a program, calling it razor is fine

Comment (a16bitsysop) [reply contrib/razor/template.py:1]: okay changed back, yes it a program for checking email hashes against a database for spam

**Pull request merged**